pom.xml

```diff
@@ -2140,7 +2146,7 @@
 
         <carbon.metrics.version>1.3.12</carbon.metrics.version>
         <!-- apache pdfbox version -->
-        <pdfbox.version>2.0.25</pdfbox.version>
+        <pdfbox.version>3.0.1</pdfbox.version>
```

Comment:
need to add a separate entry for pdfbox-io version (3.0.1.wso2v1) and change the imports accordingly